openedx/features/content_type_gating/tests/test_access.py

```diff
@@ -124,20 +124,23 @@ def _assert_block_is_gated(block, is_gated, user, course, request_factory, has_u
             assert 'content-paywall' not in content
 
     fake_request = request_factory.get('')
-    with patch('lms.djangoapps.course_api.blocks.api.is_request_from_mobile_app', return_value=False):
```

Comment:
This line was unnecessary - from a time before that utility function got simplified. No longer affects tests unless they do something special.

Unindenting this block makes the diff hard to read, but here are the real changes:
- Added `contains_gated_content` to the requested fields list
- Added an assert on the resulting field.